### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by Megha-Joshi**
Screenshot (111)

This is the Navigation component of my CL. Please review it!

### CODE REVIEW

File: Components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+}
+
+.btn-only-icon {
+    min-width: 0px;
+}
+
+/* Override Component Library */
+
+.no-margin {
+    margin: 0;
```
Comment (varun13169) [inline Components/Navigation/navigation.css:103]: try to put it in a separate util css

**Review by varun13169 — APPROVED** [review]
Code looks good, made a comment on a small issue, can be taken up as a follow up.

**Review by srishtigautam21 — APPROVED** [review]
The code is clean
good to merge

File: Components/Navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    margin: 0.5rem;
+    cursor: pointer;
+    font-weight: 400;
+}
+
+.left-navi-section :hover {
+    color: var(--accent-color);
+}
+
+
```
Comment (zoyatanweer) [inline Components/Navigation/navigation.css]: remove spacing

File: Components/Navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+.items-color {
+    color: var(--font-color);
+}
+
```
Comment (zoyatanweer) [inline Components/Navigation/navigation.css:70]: remove spacing

**Review by zoyatanweer — APPROVED** [review]
- nice use of variables in the css file
- clean code and good class naming convention
- understandable code
- good to merge

File: Components/Navigation/navigation.css (hunk 3 above)
Comment (Megha-Joshi) [reply Components/Navigation/navigation.css:70]: Done!

File: Components/Navigation/navigation.css (hunk 1 above)
Comment (Megha-Joshi) [reply Components/Navigation/navigation.css:103]: Okay will do that!

**Pull request merged**